### PR TITLE
Issue #789 Fix docs build uploads to create unwanted directories

### LINKED ISSUE #789: Docs build upload is creating unwanted directories
It seems the upload of _minishift-adoc.tar_ into the latest directory does not work as expected. There are directories created recursively - http://artifacts.ci.centos.org/minishift/minishift/docs/latest/latest/latest/latest/latest/latest/latest/

Also there are by now plenty of unwanted artefacts in http://artifacts.ci.centos.org/minishift/minishift/docs

* latest-alpha 
* latest-released	 
* minishift-adoc.tar

Let's clean this up, including the _latest_ directory which should only contain the latest _minishift-adoc.tar_.

### PR DESCRIPTION
**Pull request by budhrg**
Fix #789 

As part of test, I will be creating `latest_test` directory.

I have removed the soft linking the directory since in the remote server, it is actually 
creating a folder.

### CODE REVIEW

File: centos_ci.sh [hunk 1]
```diff
@@ -201,12 +199,13 @@ function docs_tar_upload() {
   set +x
 
   version=$(cat docs/source/variables.adoc | cut -d' ' -f2 | head -n1)
-  mkdir -p minishift/docs/$version
+  LATEST=latest_test
+  mkdir -p minishift/docs/$version minishift/docs/$LATEST
   cp docs/build/minishift-adoc.tar minishift/docs/$version/
-  ln -s $(pwd)/minishift/docs/$version minishift/docs/latest
+  cp docs/build/minishift-adoc.tar minishift/docs/$LATEST/
```
Comment (hferentschik) [inline centos_ci.sh]: So you basically create both directories (latest and version) and copy them. No symbolic link. Not quite the same, but it will do

Comment (hferentschik) [inline centos_ci.sh]: Looks like http://artifacts.ci.centos.org/minishift/minishift/docs/ is cleaned up. Could you do this yourself?

File: centos_ci.sh [hunk 2]
```diff
@@ -220,7 +219,7 @@ else
   source ~/jenkins-env # Source environment variables for minishift_ci user
   PASS=$(echo $CICO_API_KEY | cut -d'-' -f1-2)
 
-  if [[ "$JOB_NAME" = "minishift-docs" ]]; then
+  if [[ "$JOB_NAME" != "minishift-docs" ]]; then
```
Comment (hferentschik) [inline centos_ci.sh]: not equal !? Is this intended?

File: test/integration/features/basic.feature [hunk 3]
```diff
@@ -45,7 +45,7 @@ Feature: Basic
     """
 
   Scenario: User has a pre-configured set of persitence volumnes
-    When executing "oc get pv --as system:admin -o=name" retrying 5 times with wait period of 3 seconds
+    When executing "oc get pv --as system:admin -o=name" retrying 12 times with wait period of 5 seconds
```
Comment (hferentschik) [inline test/integration/features/basic.feature]: How come you changed this? Did you have issues with the test failing again? I wonder whether there is maybe a different issue after all?

File: centos_ci.sh (hunk 1 above)
Comment (budhrg) [inline centos_ci.sh]: @hferentschik , I did it manually and commented here https://github.com/minishift/minishift/issues/789#issuecomment-296201892 as well.

File: centos_ci.sh (hunk 2 above)
Comment (budhrg) [inline centos_ci.sh]: Intended as I need to check that path, will remove it once I am convinced it is fine.

**Comment by budhrg** [comment]
@hferentschik , IMO, the PR is ready and working as expected. Please review.

**Comment by hferentschik** [comment]
Merged, thanks.

**Pull request closed without merging**